packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -224,10 +224,11 @@ export class MediaRequestManager {
   // eslint-disable-next-line class-methods-use-this
   private getH264MaxMbps(mediaRequest: MediaRequest): number {
     // fallback for maxFps (not needed for maxFs, since there is a fallback already in getDegradedClientRequests)
-    const maxFps = mediaRequest.codecInfo.maxFps || CODEC_DEFAULTS.h264.maxFps;
+    const maxFps = mediaRequest.codecInfo?.maxFps || CODEC_DEFAULTS.h264.maxFps;
+    const maxFs = mediaRequest.codecInfo?.maxFs || CODEC_DEFAULTS.h264.maxFs;
```

Comment:
From types it seems that both `maxFps` and maxFs can be undefined, so I added here default value too

Reply:
the comment explains why in practice `maxFs` cannot be undefined, but it doesn't hurt to have the fallback